code/game/objects/items/pneumaticCannon.dm

```diff
@@ -68,6 +68,15 @@
 /obj/item/pneumatic_cannon/CanItemAutoclick()
 	return automatic
 
+/obj/item/pneumatic_cannon/proc/pressureSetting_to_text(pressureSetting)
```

Comment:
Same comments here, except make this `pressure_setting_to_text` as well, and the var `pressure_setting`. I know the var is pressureSetting but it should not be